docs/03-key-value-API.md

```diff
@@ -92,15 +163,64 @@ memcached_fetch(memcached_st *ptr,
 `memcached_mget`은 주어진 key 배열에 대한 value들을 조회하는 요청을 보낸다.
 keys 인자는 key pointer array이고, key_length 인자는 key length array이다.
 
-`memcached_fetch`는 mget 요청에 대한 결과를 하나씩 꺼내온다.
-error 값이 MEMCACHED_END가 될 때까지 실행하면 되며, 그 이후에 fetch를 수행하면 MEMCACHED_NOTFOUND가 반환된다.
+`memcached_fetch`는 mget 요청에 대한 결과를 하나씩 가져온다. 반환된 결과가 NULL이 아닌 경우 반드시 free 해주어야 한다. error 값이 MEMCACHED_END가 될 때까지 계속 실행하면 되며, 이후에 fetch를 수행하면 MEMCACHED_NOTFOUND가 반환된다.
+
+```c
+int arcus_kv_mget(memcached_st *memc)
```

Comment:
https://docs.oracle.com/cd/E88353_01/html/E37842/memcached-get-3libmemcached.html

memcached_mget 매뉴얼을 참고하세요.

Reply:
> - https://docs.oracle.com/cd/E88353_01/html/E37842/memcached-fetch-3libmemcached.html
Deprecated  since  version  0.50:  Use  `memcached_fetch_result()` instead.
> - https://docs.oracle.com/cd/E88353_01/html/E37842/memcached-mget-3libmemcached.html
To  retrieve data after a successful execution of `memcached_mget()`, you will need to call `memcached_fetch_result()`. You  should  continue  to call  this  function  until it returns a `NULL` (i.e. no more values). If you need to quit in the middle of a `memcached_mget()` call, you can execute a `memcached_quit()`, those this is not required.

참고로, 문서 기준으로 `memcached_fetch()`는 deprecated 되었고, `memcached_fetch_result()`를 기준으로 설명하고 있습니다.

Reply:
> 문서 기준으로 `memcached_fetch()`는 deprecated 되었고, `memcached_fetch_result()`를 기준으로 설명하고 있습니다.

이유가 무엇이라고 생각하나요?
우리는 memcached_fetch_result() 함수를 사용해야 하지 않나요?

Reply:
0.50 changelog를 보면 아래와 같은 표현이 있는데, 이와 연관이 있을 것으로 추측합니다.
> memcached_fetch_result() now uses the internal result when available (about 25 to 50% faster).

추가로, commit 이력을 보면 [memcached_fetch()](https://github.com/awesomized/libmemcached/commit/ecc7382f48857cb22e74ffa9496f9a31e956bde3#diff-253326ad3f7a858eb6ed15900a6ceb567a83dc60a47bb380db75ef579d76bbce)는 문서가 추가될 때부터 deprecated 상태로 추가되었습니다.


정확한 이유는 모르겠지만, 어찌 되었든 `memcached_fetch_result()` 사용하는 것이 나을 것으로 생각합니다.